posydon/popsyn/normalized_pop_mass.py

```diff
@@ -19,9 +19,8 @@ def initial_total_underlying_mass(df=None, df1=None, df2=None, **kwargs):
 
     Parameters
     ----------
-    df : DataFrame
-        Data frame of a population class. If nothing is provided the code will
-        sample initial conditions.
+    df,df1,df2 : float
+        Total simulated mass, simulated mass of binary systems and simulated mass of single stars, respectively.
```

Comment:
If they aren't data frames anymore, I think we should consider to rename them.
I'd suggest to call them `simulated_mass`, `simulated_mass_single`, and `simulated_mass_binaries`, respectively.